README.md

```diff
@@ -23,45 +23,39 @@
 </div>
 
-## What is MapReader?
+# What is MapReader?
+
+MapReader is an end-to-end computer vision (CV) pipeline for analyzing and exploring large collections of maps and images. 
+
+MapReader was developed in the [Living with Machines](https://livingwithmachines.ac.uk/) project to analyze large collections of historical maps but is a _**generalisable**_ computer vision pipeline which can be applied to _**any images**_ in a wide variety of domains. See [Gallery](#gallery) for some examples.
+
+Refer to each tutorial/example in the [use cases](#use-cases) section for more details on MapReader's relevant functionalities for <ins>non-geospatial</ins> and <ins>geospatial</ins> images and how MapReader can help to analyze large image datasets.
 
-MapReader is an end-to-end computer vision (CV) pipeline for analyzing large collections of images/maps. It has two main components: preprocessing/annotation and training/inference:
+MapReader has two main components: preprocessing/annotation and training/inference as shown in this figure:
 
 <p align="center">
   <img src="./figs/MapReader_pipeline.png" 
         alt="MapReader pipeline" width="70%" align="center">
 </p>
 
-MapReader provides a set of tools to:
+It provides a set of tools to:
 
-- **load** images/maps stored locally or **retrieve** maps via web-servers (e.g., tileservers which can be used to retrieve maps from OpenStreetMap (OSM), the National Library of Scotland (NLS), or elsewhere). :warning: Refer to the [credits and re-use terms](#credits-and-re-use-terms) section if you are using digitized maps or metadata provided by NLS. 
-- **preprocess** images/maps (e.g., divide them into patches, resampling the images, removing borders outside the neatline or reprojecting the map).
-- annotate images/maps or their patches (i.e. slices of an image/map) using an **interactive annotation tool**.
+- **load** images or maps stored locally or **retrieve** maps via web-servers (e.g., tileservers which can be used to retrieve maps from OpenStreetMap (OSM), the National Library of Scotland (NLS), or elsewhere). :warning: Refer to the [credits and re-use terms](#credits-and-re-use-terms) section if you are using digitized maps or metadata provided by NLS. 
+- **preprocess** images or maps (e.g., divide them into patches, resampling the images, removing borders outside the neatline or reprojecting the map).
+- annotate images or maps or their patches (i.e. slices of an image or map) using an **interactive annotation tool**.
 - **train, fine-tune, and evaluate** various CV models.
-- **predict** labels (i.e., model inference) on large sets of images/maps.
+- **predict** labels (i.e., model inference) on large sets of images or maps.
 - Other functionalities include:
     - various **plotting tools** using, e.g., *matplotlib*, *cartopy*, *Google Earth*, and [kepler.gl](https://kepler.gl/).
     - compute mean/standard-deviation **pixel intensity** of image patches.
 
```

Comment:
MapReader is a groundbreaking interdisciplinary tool that emerged from a specific set of geospatial historical research questions. It was inspired by methods in biomedical imaging and contemporary mapping, which were adapted for annotation and use by historians. The success of the tool subsequently generated interest from plant phenotype researchers working with large image datasets, and so MapReader is an example of cross-pollination between the humanities and the sciences made possible by reproducible data science.

Reply:
@dcsw2 Thanks. Done: https://github.com/Living-with-machines/MapReader/pull/38/commits/2d0d3c2c9ed2e3752d9e0e5375c95f7e5d36c49c

Reply:
Hi @kasra-hosseini could you please incorporate katie's suggestion of 'geographic information science' for 'contemporary mapping'? I don't know how to access the lates text version.

Reply:
Is this OK? https://github.com/Living-with-machines/MapReader/pull/38/commits/a7eb4b6f28b90000e01e58e1b61f95f1110ed893